### ISSUE TITLE
Error when fetch string like " < text > "

### ISSUE THREAD

**Issue by gcasasi** [fiduswriter/simple-datatables#237]
Hi, I want to report a bug, I detected in my installation and I tryed to reproduce with demo (https://fiduswriter.github.io/simple-datatables/demos/18-fetch-api/),. 

For example, if I change value of "Drink" and put something like " < text > ", 

```
{
    "ID": 1,
    "Drink": "< text >",
    "Caffeinated": false,
    "Price": 4,
    "Profit": 0
  }
```

I get this error

`simple-datatables.js:3 Uncaught (in promise) DOMException: Failed to execute 'createElement' on 'Document': The tag name provided ('') is not a valid name.
    at s (https://cdn.jsdelivr.net/npm/simple-datatables@latest:7:1337)
    at https://cdn.jsdelivr.net/npm/simple-datatables@latest:7:1499
    at Array.forEach (<anonymous>)
    at s (https://cdn.jsdelivr.net/npm/simple-datatables@latest:7:1473)
    at https://cdn.jsdelivr.net/npm/simple-datatables@latest:7:1499
    at Array.forEach (<anonymous>)
    at s (https://cdn.jsdelivr.net/npm/simple-datatables@latest:7:1473)
    at https://cdn.jsdelivr.net/npm/simple-datatables@latest:7:1499
    at Array.forEach (<anonymous>)
    at s (https://cdn.jsdelivr.net/npm/simple-datatables@latest:7:1473)`

**Comment by johanneswilm** [comment]
Yes, it's interpreted as html and there us no html element with that name. So you have to html escape your "<"s and ">"s.

**Comment by johanneswilm** [comment]
Hey,
yes sure. Normally this is not a problem as you will need to HTML escape all user input before putting it out on the web anyway because else the users can add their own HTML on your website. You can have that escape function run in JavaScript before displaying the table rather than on your server.
 
 Alternatively, you can turn of SDT's data conversion mechanism by setting `dataConvert: false` in the options. That means that you will have to provide the data in the native format that SDT uses internally. Each cell is an object which contains at least the key `data` and possible also `text` and `order`, like this:
 
 ```javascript
 const data = [
    [ // one row of the table
        {data: '<SIMPLE>', order: 1}, {data: 12, text: "Twelve"}
    ],
    [ // another row of the table
        {data: '<COMPLEX>', order: 2}, {data: 13, text: "Thirteen"}
    ]
]
 ```
 See the full description in the manual: https://fiduswriter.github.io/simple-datatables/documentation/API#data

**Comment by johanneswilm** [comment]
Hey @gcasasi there is a fix in version 6.0.8 that has just been released.
